restic/internal/cat.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from restic.internal import command_executor
+
+
+class Cat:
+
+    def __init__(self, restic_base_command):
+        self.base_command = restic_base_command
+
+    def _run(self, subcommand, id_=None, json_encode=True, binary_mode=False):
```

Comment:
id() is an internal Python function. [PEP8](https://peps.python.org/pep-0008/#descriptive-naming-styles) defines:

> single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.